src/modules/taskList.js

```diff
@@ -6,15 +6,47 @@ export default class {
     this.list = list;
   }
 
+  regenerateValues(datalist) {
+    const takslist = [];
+    datalist.forEach((entry) => {
+      takslist.push(new Task(
+        entry.value,
+        entry.index,
+        entry.completed,
+      ));
+    });
+
+    this.list = takslist;
+  }
+
+  exportListValues() {
+    const datalist = [];
+    this.list.forEach((task) => {
+      datalist.push({
+        value: task.value,
+        index: task.index,
+        completed: task.completed,
+      });
+    });
+
+    return datalist;
+  }
+
   add(value) {
-    this.list.push(new Task(value, this.list.length));
+    const task = new Task(value, this.list.length);
+    this.list.push(task);
+    return task;
   }
```

Comment:
- [ ] Your app is working great, nice job! However, as required by the activity, the indexes in the array should start from 1:

- By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).

Your array:
<img width="1190" alt="Screen Shot 2022-07-16 at 12 37 55 AM" src="https://user-images.githubusercontent.com/83717285/179341634-64972538-db2f-4f6a-a226-671dec3cbeb5.png">

It would be great to make make the indexes start from 1.

Reply:
I have fixed the index property! Now it should be starting from 1 as suggested